test/evaluation/uriEvaluationHandler.spec.ts

```diff
@@ -45,21 +45,21 @@ describe('evaluate', () => {
 
   it('should generate 6 error result fo this test case.', () => {
     const pdSchema: PresentationDefinition = getFile('./test/dif_pe_examples/pd/input_descriptor_filter_examples.json').presentation_definition;
-    const vpSimple = getFile('./test/dif_pe_examples/vp/vp_general.json');
+    const vpSimple: VerifiablePresentation = getFile('./test/dif_pe_examples/vp/vp_general.json');
     const evaluationClient: EvaluationClient = new EvaluationClient();
     const evaluationHandler = new UriEvaluationHandler(evaluationClient);
-    evaluationHandler.handle(pdSchema, new VP(vpSimple));
+    evaluationHandler.handle(pdSchema, vpSimple);
     const errorResults = evaluationClient.results.filter(result => result.status === Status.ERROR);
     expect(errorResults.length).toEqual(6);
   });
 
   it('should generate 5 error result and 1 info.', () => {
     const pdSchema: PresentationDefinition = getFile('./test/dif_pe_examples/pd/input_descriptor_filter_examples.json').presentation_definition;
-    const vpSimple = getFile('./test/dif_pe_examples/vp/vp_general.json')
+    const vpSimple: VerifiablePresentation = getFile('./test/dif_pe_examples/vp/vp_general.json')
```

Comment:
we had to move these to TypeScript files from json files.

ps. all json example should be moved to ts.